src/index/testing.rs

```diff
@@ -128,7 +128,20 @@ impl Context {
     let balances = balances.as_mut();
     balances.sort_by_key(|(outpoint, _)| *outpoint);
 
-    for (_, balances) in balances.iter_mut() {
+    let mut burned_balances = 0;
```

Comment:
I would just call this burned. It's not really "balances" because it's just a number. Balances so named because it's a map.